src/ripe_rainbow/domain/logic/ripe_pulse.py

```diff
@@ -21,6 +21,21 @@ def logout_wait(self, redirect_url = None):
         self.logout()
         self.waits.redirected_to(redirect_url)
 
+    def click_order(self, order):
+        self.interactions.click_when_possible(".table .id a[href^='%s']" % self.order_url(order, absolute = False))
+        self.waits.redirected_to(self.order_url(order))
+        self.waits.text(".title", "Order #%s" % order)
+
+    def click_report(self, order):

```

Comment:
This does not make sense there's no practical value in here

Reply:
The point was in the future adding an assertion checking that the report page was opened. Right now I didn't do it since it's a new page and we don't have base functionality to check that, but if we use this function in the tests already in the future we gain the assertion for free without changing anything.

Reply:
OK, let's wait for the future then. Once the future happens we make the change ;)